src/core/init/load_on_media_source.ts

```diff
@@ -174,6 +174,12 @@ export default function createMediaSourceLoader({
               handleDiscontinuity(gap[1], mediaElement);
             }
             return EMPTY;
+          case "needs-source-buffer-flush":
+            if (mediaElement.currentTime + 0.001 < mediaElement.duration) {
+                log.info("Init: Flushing buffers by performing a small seek.");
+                mediaElement.currentTime += 0.001;
+              }
```

Comment:
Indentation is a little inconsistent here.
Also should we do nothing if that condition is false? (we may not want to seek behind because the stream has most likely ended, but we can perhaps seek in place)

Reply:
Yea, fixed the indentation, thanks;

However, what the point of seeking in place?

If I understand right, it would mean:
```
mediaElement.currentTime = mediaElement.currentTime;
```
but what's the point of doing this?
?

Reply:
> fixed the indentation
Not that I can see

> but what's the point of doing this?
Seeking to the same position will still flush the buffers on most platforms I think (we're just a little less sure that it does).

The real issue I'm afraid of here is if the current player is 0.0005 seconds behind the end here, it won't flush and possibly be stuck before the ending.

Reply:
oh ok, yes since we are checking `0.001` :+1:  

Do you see any problem by setting the `currentTime` of the media element to the overall `duration` then?

Reply:
> Do you see any problem by setting the currentTime of the media element to the overall duration then?

Yeah why not?